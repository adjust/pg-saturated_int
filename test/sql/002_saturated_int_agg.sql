CREATE TABLE sat_agg (id1 saturated_int, id2 saturated_int);

INSERT INTO sat_agg
SELECT i, 1 FROM pg_catalog.generate_series(2147483645::bigint, 2147483650::bigint) g(i);

SELECT * FROM sat_agg;

-- Test aggregate functions
SELECT sum(id1), sum(id2) FROM sat_agg;
