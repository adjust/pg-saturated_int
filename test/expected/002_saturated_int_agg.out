CREATE TABLE sat_agg (id1 saturated_int, id2 saturated_int);
INSERT INTO sat_agg
SELECT i, 1 FROM pg_catalog.generate_series(2147483645::bigint, 2147483650::bigint) g(i);
SELECT * FROM sat_agg;
    id1     | id2 
------------+-----
 2147483645 | 1
 2147483646 | 1
 2147483647 | 1
 2147483647 | 1
 2147483647 | 1
 2147483647 | 1
(6 rows)

-- Test aggregate functions
SELECT sum(id1), sum(id2) FROM sat_agg;
    sum     | sum 
------------+-----
 2147483647 | 6
(1 row)

